galasa-ui/src/components/users/UsersTable.tsx

```diff
@@ -139,6 +150,22 @@ export default function UsersTable({ usersListPromise, currentUserPromise }: Use
 
   };
 
+  const deleteUser = async (userNumber: string) => {
+    try {
+      const response = await deleteUserFromService(userNumber);
+      if (response.status === 204) {
```

Comment:
Same comment about status codes in server actions - if they're not needed, this code could be simplified